module/zcommon/zfeature_common.c

```diff
@@ -553,6 +553,17 @@ zpool_feature_init(void)
 	    "com.datto:resilver_defer", "resilver_defer",
 	    "Support for defering new resilvers when one is already running.",
 	    ZFEATURE_FLAG_READONLY_COMPAT, ZFEATURE_TYPE_BOOLEAN, NULL);
+	{
+	static const spa_feature_t zilsaxattr_deps[] = {
+		SPA_FEATURE_EXTENSIBLE_DATASET,
+		SPA_FEATURE_NONE
+	};
+	zfeature_register(SPA_FEATURE_ZILSAXATTR,
+	    "org.zfsonlinux:zilsaxattr", "zilsaxattr",
+	    "Zil sa xattr.",
+	    ZFEATURE_FLAG_READONLY_COMPAT | ZFEATURE_FLAG_PER_DATASET,
+	    ZFEATURE_TYPE_BOOLEAN, zilsaxattr_deps);
+	}
```

Comment:
I don't see where this gets activated.  I think this would have to be ZFEATURE_FLAG_ACTIVATE_ON_ENABLE (and not ZFEATURE_FLAG_PER_DATASET).

Reply:
yes. Changed it.